controllers/openstackmachine_controller.go

```diff
@@ -336,9 +337,14 @@ func (r *OpenStackMachineReconciler) reconcileNormal(ctx context.Context, logger
 		openStackMachine.Status.Ready = true
 	case infrav1.InstanceStateBuilding:
```

Comment:
This should be deleted because there is not `BUILDING` status.

Reply:
I don't even notice that, will check again, thanks for the info~